transform/rtcalls.go

```diff
@@ -49,7 +49,7 @@ func OptimizeStringToBytes(mod llvm.Module) {
 					// There is a store to the byte slice. This means that none
 					// of the pointer uses can't be propagated.
 					canConvertPointer = false
-					continue
+					break
```

Comment:
Note that this actually breaks from the switch statement, not from the for loop. So it only avoids a single `append` statement.